### PR TITLE
Update configuration properties for aad and aadb2c.

### PR DESCRIPTION
**Pull request by chenrujun**
As title.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @saragluna , please help to review this PR when you have time.

File: docs/src/main/asciidoc/spring-security-support.adoc [hunk 1]
```diff
@@ -25,26 +25,30 @@ The `spring-cloud-azure-starter-active-directory` (`aad-starter` for short) prov
...
-|*spring.cloud.azure.active-directory*.resource-server.principal-claim-name |Principal claim name. Default value is "sub".
-|*spring.cloud.azure.active-directory*.user-group.allowed-group-ids |Users' group id can be use in `@PreAuthorize(&quot;hasRole(&#39;ROLE_group_id_1&#39;)&quot;)`. Not all group id will take effect, only group id configured in this property will take effect. If this property's value is `all`, then all group id will take effect.
-|*spring.cloud.azure.active-directory*.user-group.allowed-group-names |Users' group name can be use in `@PreAuthorize(&quot;hasRole(&#39;ROLE_group_name_1&#39;)&quot;)`. Not all group name will take effect, only group names configured in this property will take effect.
-|*spring.cloud.azure.active-directory*.user-group.use-transitive-members |Whether to use `/v1.0/me/transitiveMemberOf` instead of  `/v1.0/me/memberOf` to get groups.
-|*spring.cloud.azure.active-directory*.user-name-attribute |Decide which claim to be principal's name. 
+|Name |Default |Description
```
Comment (saragluna) [inline docs/src/main/asciidoc/spring-security-support.adoc:28]: The two tables are not rendering correctly 
image

**Review by saragluna — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/spring-security-support.adoc (hunk 1 above)
Comment (chenrujun) [reply docs/src/main/asciidoc/spring-security-support.adoc:29]: Fixed: 
image

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**